website/blog/2024-04-22-release-0.74.md

```diff
@@ -0,0 +1,402 @@
+          setState1(true);
+       }}>
+      <View
+         onLayout={() => {
+          // When this event is executed, state1's new value is no longer observable here.
+          setState2(true);
+        }}>
+      </View>
+    </View>
+  );
+}
+```
+
+In 0.74, `setState1` and `setState2` updates are batched together. This change is [expected behavior in React](https://react.dev/learn/queueing-a-series-of-state-updates#react-batches-state-updates) and allows for less re-renders.
+
+:::danger
+This change **may break code** that has relied on un-batched state updates. You'll need to refactor this code to use [updater functions](https://react.dev/learn/queueing-a-series-of-state-updates#updating-the-same-state-multiple-times-before-the-next-render) or equivalent.
+:::
+
+### Yarn 3 for New Projects
+
+[Yarn 3](https://yarnpkg.com/blog/release/3.0) is now the default JavaScript package manager for new projects initialized with React Native Community CLI.
+
+Yarn 3.x will be used with `nodeLinker: node-modules`, a mode providing compatibility with React Native libraries. This replaces Yarn Classic (1.x, deprecated) as the previous default. To upgrade Yarn version inside your existing app you can follow this [guide](https://yarnpkg.com/migration/guide).
+
+```sh
+$ yarn --help
+━━━ Yarn Package Manager - 3.6.4 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  $ yarn <command>
+```
+
+The Community CLI also supports initializing projects with other package managers via the `--pm` flag ([read more](https://github.com/react-native-community/cli/blob/main/docs/init.md)).
+
+## Breaking Changes
+
+### Android SDK minimum bump (Android 6.0)
+
+React Native 0.74 has a minimum Android SDK version requirement of 23 (Android 6.0). Previously, this was Android 5.0 (API 21). See our context for this change [here](https://github.com/react-native-community/discussions-and-proposals/discussions/740).
+
+#### Bonus: Android app size reduction
+
+The minimum SDK bump, together with several improvements at our native build, allowed us to greatly reduce the app size on user devices.
+
+For example a newly created app with React Native 0.74 occupies ~13% less space on user device, resulting in ~4MB saved on device.
+
+![Side-by-side comparison of a new React Native app in the Android system storage view](/blog/assets/0.74-android-app-size.jpg)
+
+### Removal of deprecated `PropTypes`
+
+Before 0.74, React Native was continuing to ship with `PropTypes`, an API that had been deprecated since React 15.5 in 2017! The presence of this deprecated code incurs costs in app size and memory overhead for apps that follow best practices.
+
+Starting in 0.74, React Native will no longer include the following `PropTypes` properties: `Image.propTypes`, `Text.propTypes`, `TextInput.propTypes`, `ColorPropType`, `EdgeInsetsPropType`, `PointPropType`, `ViewPropTypes` (see [commit](https://github.com/facebook/react-native/commit/228cb80af9ded20107f3c7a30ffe00e24471bfeb)).
```

Comment:
@cortinico @yungsters Would it be fair to say (as a summary, preceding this) that 0.74 removes _**all built-in `PropTypes`**_ from React Native?

Reply:
@huntie Yes, that is accurate.